app/src/main/res-screen/welcome/layout/fragment_welcome_screen.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".ui.WelcomeFragment">
```

Comment:
Use `Option + Command + L` to format XML code. It should be pretty as well as general codebase. Empty lines between views...